### ISSUE TITLE
Which project to fork?

### ISSUE THREAD

**Issue by kutakmir** [kutakmir/OpenFirebaseChat#1]
Let's discuss which project to base this library on!

My solution is in the developer branch:
https://github.com/kutakmir/OpenFirebaseChat/tree/develop

**Comment by kilitbilgi** [comment]
Hi, we can fork your repo, what could we add, what is your base features on this repo, what do we need to add, I think, we can do UI flexibility, new features like group chat, typing so on.

**Comment by kilitbilgi** [comment]
I think, we can use async node for chat ui.

**Comment by kilitbilgi** [comment]
Need to add color scheme constants.

**Comment by mrcflorian** [comment]
What's the motivation of starting from scratch rather than contributing to the repo that already has the most stars / popularity / features? I love the idea of creating an ultimate chat app, but why start from scratch?

**Comment by bensmiley** [comment]
@mrcflorian I agree. I have spent the last 5 years developing an open source [Firebase powered chat app](https://github.com/chat-sdk). Is it perfect? no. But it supports iOS, Android and has a web based version. We already have over 1k stars on our Android repository and a large community of users and contributors. We also have 3 people working full time on the library.

Some of the features you suggest are things we'd love to add to our library but we just don't have the time. If you are keen to get into instant messaging, why not contribute to your project. I'd be happy to provide any support you need to get started.
